reflex/utils/china.py

```diff
@@ -0,0 +1,27 @@
+"""This module contains utility functions for checking if the current IP address is in China."""
+
+from typing import Union
+
+import requests
+
+
+def _is_in_china() -> Union[bool, None]:
+    """Check if the current IP address is in China.
+
+    Returns:
+        bool: True if the IP address is in China, False otherwise.
+        None: If an error occurred.
+    """
+    try:
+        response = requests.get(f"http://ip-api.com/json")
```

Comment:
We should use `httpx` to make any requests.

According to `pyproject.toml` reflex does not directly depend on `requests`, so if it happens to be installed, then we're getting it as a transitive dependency and cannot rely on it being in the environment (for example, if the dependency that brings it in transitively switches to `httpx` or `urllib3` or something else)